doc/changes/changes_0.6.1.md

```diff
@@ -2,7 +2,7 @@
 
 Code name: 0.6.1: Upgrade dependencies
 
-This release upgrades dependencies and reduces the number of runtime dependencies, fixing [CVE-2022-21724](https://ossindex.sonatype.org/vulnerability/0f319d1b-e964-4471-bded-db3aeb3c3a29?component-type=maven&component-name=org.postgresql.postgresql&utm_source=ossindex-client&utm_medium=integration&utm_content=1.1.1) in the PostgreSQL JDBC driver.
+This release upgrades dependencies and reduces the number of runtime dependencies, fixing [CVE-2022-21724](https://ossindex.sonatype.org/) in the PostgreSQL JDBC driver.
```

Comment:
Hm, this is the old changelog. If you change it now, it will be inconsistent with the GH release notes.

Reply:
Yes, but the link is broken and the link checker complains about it.

Reply:
We discussed this, adequate solution for now, there's not really any info loss.